resources/views/lessons/show.blade.php

```diff
@@ -129,9 +129,9 @@
                             </div>
                         </li>
                         <li class="list-group-item d-flex justify-content-center">
-                            <form action="{{ route('courses.leave', $course) }}">
+                            <form method="post" action="{{ route('courses.users.destroy', [$course, Auth::user()]) }}">
```

Comment:
lại nhanh nhảu rồi, ở đây đã sửa đâu

Reply:
Nếu em không truyền vào nó sẽ báo lỗi:
Missing required parameter for [Route: courses.users.destroy] [URI: courses/{course}/users/{user}]